### PR TITLE
Remove adapters deprecation warning

### PR DESCRIPTION
**Pull request by fabianrbz**
Hi guys,

This pull request removes the following [deprecation warning](https://github.com/colszowka/simplecov/blob/fee9dcf1f990a57503b0d518d9844a7209db4734/lib/simplecov/configuration.rb#L124) from simplecov.

The new way is using [profiles](https://github.com/colszowka/simplecov#profiles). The warning comes from the gem 'simplecov-gem-adapter', which haven't been updated for the last 4 years.
In this pull request l removed the gem dependency and added a new profile with some of the settings that were in the gem. l thought it was better this way -instead of making a PR to the gem- because this way, we remove a dependency and it will be easier to customize the profile in the future.
What do you think?

Cheers!

### CODE REVIEW

File: lib/simplecov_custom_profile.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+require 'simplecov'
+
+SimpleCov.profiles.define 'gem' do
+  add_filter '/test/'
+  add_filter '/features/'
+  add_filter '/autotest/'
+
+  add_group 'Binaries', '/bin/'
+  add_group 'Libraries', '/lib/'
+end
```
Comment (parkr) [inline lib/simplecov_custom_profile.rb]: Let's put this file in `test`.

File: test/helper.rb [hunk 2]
```diff
@@ -1,5 +1,4 @@
-require 'simplecov'
-require 'simplecov-gem-adapter'
+require 'simplecov_custom_profile'
```
Comment (parkr) [inline test/helper.rb]: Please use `File.expand_path` to require this file.

**Comment by parkr** [comment]
Great!!

**Comment by fabianrbz** [comment]
@parkr thanks! what happened to the comments you made? were they implemented?

**Comment by parkr** [comment]
> @parkr thanks! what happened to the comments you made? were they implemented?

Yep! fbe52bca85b66ad89040ecd1519dc9021ab5a532

**Comment by fabianrbz** [comment]
sweet! thanks @parkr!

**Pull request merged**